hpx/runtime/threads/policies/thread_queue_mc.hpp

```diff
@@ -16,6 +16,7 @@
 #include <hpx/errors.hpp>
 #include <hpx/functional/function.hpp>
 #include <hpx/runtime/threads/policies/lockfree_queue_backends.hpp>
+#include <hpx/runtime/threads/policies/maintain_queue_wait_times.hpp>
```

Comment:
Should this also `#include <hpx/runtime/threads/policies/deadlock_detection.hpp>`?

Reply:
Yes, thanks!